kv/map.go

```diff
@@ -98,9 +97,6 @@ func (me *MapEditor) Get(key types.Value) types.Value {
 // be parsed from canonical json, otherwise we might parse two different
 // values for the same canonical json.
 func (me *MapEditor) Set(key types.String, value types.Value) error {
-	if key == "" {
```

Comment:
We could potentially wire through the version to this place. I decided it wasn't worth it.